packages/devtools/client/src/utils/service-agent.ts

```diff
@@ -1,12 +1,32 @@
+import { hc } from 'hono/client';
+import type { AppType } from '@/service-worker';
+
 export interface ModifyHeaderRule {
   id: string;
   test?: string;
   key: string;
   value: string;
 }
 
-export interface ServiceStatus {
-  href: string;
-  version: string;
-  rules: ModifyHeaderRule[];
-}
+export const SERVICE_SCRIPT = '/sw-devtools.js';
+
+export const swClient = hc<AppType>('https://__devtools-api.modernjs.dev');
```

Comment:
`https://__devtools-api.modernjs.dev` is also used in `service-worker.ts`, we can use a constant